packages/wallets/src/components/Base/WalletDropdown/WalletDropdown.tsx

```diff
@@ -98,16 +98,20 @@ const WalletDropdown: React.FC<TProps> = ({
                     onKeyUp={() => setShouldFilterList(true)}
                     placeholder={reactNodeToString(label)}
                     readOnly={variant !== 'comboBox'}
-                    renderLeftIcon={() => icon}
-                    renderRightIcon={() => (
-                        <button
-                            className={classNames('wallets-dropdown__button', {
-                                'wallets-dropdown__button--active': isOpen,
-                            })}
-                        >
-                            <ArrowIcon />
-                        </button>
-                    )}
+                    renderLeftIcon={icon ? () => icon : undefined}
+                    renderRightIcon={
+                        icon
```

Comment:
Do we need this check as the icon prop refers to the left icon? The right icon is excusively for `<ArrowIcon />`

Reply:
Oh you're right

Reply:
Have reverted that part